lib/ui/pointer.dart

```diff
@@ -61,7 +89,25 @@ enum PointerDeviceKind {
   /// A pointer device with a stylus that has been inverted.
   invertedStylus,
 
-  /// A touch-based pointer device with an indirect surface.
+  /// Gestures from a touch-based pointer device with an indirect surface.
+  ///
+  /// On supporting platforms, when the user is operating on a physical
```

Comment:
Here's a suggestion for an alternate wording: 

When the user makes zoom, pan, scroll or rotate gestures with a physical
trackpad, supporting platforms dispatch events with kind [trackpad].